tests/subsys/fs/fs_api/src/test_fs_dir_file.c

```diff
@@ -191,6 +191,9 @@ void test_unmount(void)
 	TC_PRINT("unmount a file system has no unmount functionality\n");
 	ret = fs_unmount(&test_fs_mnt_no_op);
 	zassert_not_equal(ret, 0, "Unmount a fs has no unmount functionality");
+	/* assign a unmount interface to test_fs_mnt_no_op to unmount it */
+	null_fs.unmount = temp_fs.unmount;
```

Comment:
Comment claims assignment to `test_fs_mnt_no_op` but assignment is done to `null_fs`

Reply:
updated.